docs/input/docs/usage/ci.md

```diff
@@ -29,3 +29,9 @@ GitVersion into an Azure DevOps build pipeline.
 
 [gittools-actions]: https://github.com/marketplace/actions/gittools
 [gittools-task]: https://marketplace.visualstudio.com/items?itemName=gittools.gittools
+
+## GitLab CI Pipelines
```

Comment:
Please allow all link references to be placed together at the bottom of the file by moving this section up.